clickhouse/base/wire_format.cpp

```diff
@@ -99,4 +100,83 @@ bool WireFormat::SkipString(InputStream& input) {
     return false;
 }
 
+const char quoted_chars[] = {'\0', '\b', '\t', '\n', '\'', '\\'};
+
+inline const char* find_quoted_chars(const char* start, const char* end) {
+    while (start < end) {
+        char c = *start;
+        for (unsigned i = 0; i < sizeof(quoted_chars); i++) {
+            if (quoted_chars[i] == c) return start;
+        }
+        start++;
+    }
+    return nullptr;
+}
+
+void WireFormat::WriteQuotedString(OutputStream& output, std::optional<std::string_view> opt) {
+    if (!opt) //NULL
+    {
+        WriteVarint64(output, 5);
```

Comment:
Please don't use magic constants... it would be nicer to have a constant `NULL_VALUE_REPRESENTATION` pointing to a `"'\\\\N'"` string. And then do `strlen`/`std::string_view::size()`/whatever...